### ISSUE TITLE
Error TS1005: ',' expected when compile with tsc 

### ISSUE THREAD

**Issue by dr15x86** [Level/abstract-leveldown#206]
Hello. I try using this project with Typescript Version 2.7.2

But compilation failed with errors 

    node_modules/abstract-leveldown/index.d.ts:43:27 - error TS1005: ',' expected.
    
    43 export type Batch<K=any, V?=any> = PutBatch<K, V> | DelBatch<K>
                                 ~
    
    
    node_modules/abstract-leveldown/index.d.ts:43:28 - error TS1139: Type parameter declaration expected.
    
    43 export type Batch<K=any, V?=any> = PutBatch<K, V> | DelBatch<K>
                              ~

How reproduce

```bash
mkdir tst_abstract-leveldown/
cd tst_abstract-leveldown/
npm init -f
npm install --save typescript abstract-leveldown
./node_modules/.bin/tsc --init
```

create file test.ts with content
```typescript
import * as ald from 'abstract-leveldown'

console.log(ald)
```

and run compile

```bash
./node_modules/.bin/tsc --pretty
```

Versions tools:

    tsc Version 2.7.2
    abstract-leveldown 4.0.2
    NodeJS v9.5.0
    npm 5.6.0

**Comment by vweevers** [comment]
We're working on a fix in #204.

**Comment by vweevers** [comment]
Fix published in 4.0.3.
